runtime/laos/src/configs/treasury.rs

```diff
@@ -0,0 +1,95 @@
+use super::collective_council::CouncilCollective;
+use crate::{
+	currency::UNIT, weights, AccountId, Balance, Balances, BlockNumber, Permill, Runtime,
+	RuntimeEvent, Treasury,
+};
+use frame_support::{
+	parameter_types,
+	traits::{
+		tokens::{PayFromAccount, UnityAssetBalanceConversion},
+		EitherOfDiverse,
+	},
+	PalletId,
+};
+use frame_system::EnsureRoot;
+#[cfg(not(feature = "fast-mode"))]
+use parachains_common::DAYS;
+#[cfg(feature = "fast-mode")]
+use parachains_common::MINUTES;
+use sp_runtime::traits::IdentityLookup;
+
+#[cfg(feature = "fast-mode")]
+const TREASURY_SPENDING_PRERIOD: BlockNumber = 5 * MINUTES;
+#[cfg(not(feature = "fast-mode"))]
+const TREASURY_SPENDING_PRERIOD: BlockNumber = 7 * DAYS;
+
+parameter_types! {
+	pub const ProposalBond: Permill = Permill::from_percent(5);
+	pub const ProposalBondMinimum: Balance = 100 * UNIT;
+	pub const SpendPeriod: BlockNumber = TREASURY_SPENDING_PRERIOD;
+	pub const MaxApprovals: u32 = 100;
+	pub const TreasuryId: PalletId = PalletId(*b"py/trsry");
+	pub const PayoutPeriod: BlockNumber = 5;
+	pub TreasuryAccount: AccountId = Treasury::account_id();
+}
+
+type ApproveOrigin = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 1>,
+>;
+type RejectOrigin = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 2>,
+>;
+
+impl pallet_treasury::Config for Runtime {
+	type AssetKind = ();
+	type ApproveOrigin = ApproveOrigin;
+	type BalanceConverter = UnityAssetBalanceConversion;
+	type Beneficiary = AccountId;
+	type BeneficiaryLookup = IdentityLookup<Self::Beneficiary>;
```

Comment:
if the purpose of `()` is to indicate that we don't want to burn anything it might be more convenient to use `Permill::zero();` instead